.github/workflows/upload-cdn.yml

```diff
@@ -0,0 +1,62 @@
+name: Upload (CDN)
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+      folder:
+        required: true
+        type: string
+      trigger_type:
+        required: true
+        type: string
+
+jobs:
+  prepare:
+    name: Prepare
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Download all bundles
+      uses: actions/download-artifact@v4
```

Comment:
I'm don't think we have any artifacts as `gh` is used to directly upload bundles in the release assets.